tsdb/index/tsi1/partition.go

```diff
@@ -489,7 +489,11 @@ func (p *Partition) retainFileSet() *FileSet {
 }
 
 // FileN returns the active files in the file set.
-func (p *Partition) FileN() int { return len(p.fileSet.files) }
+func (p *Partition) FileN() int {
```

Comment:
I assume we're backporting this?

Reply:
This is the forward port - already in 1.10 and master-1.x